tests/ReflectionToolsTest.php

```diff
@@ -19,6 +19,85 @@ public function testGetMethodsDoesNotReturnStaticMethods()
         $this->assertCount(0, $methods);
     }
 
+    public function testGetReflectionFunction()
+    {
+        $reflectionFunc = function()
+        {
+            return 'reflectedFunction';
+        };
+        $functions = (new ReflectionTools)->getReflectionFunction($reflectionFunc);
+
+        $this->assertCount(0, $functions->getParameters());
+    }
+
+    public function testGetFunctionParameterTypesShouldReturnEmptyArray()
+    {
+        $functions = (new ReflectionTools)->getFunctionParameterTypes(new \ReflectionFunction('Brick\Reflection\Tests\reflectedFunc'));
+
+        $this->assertCount(0, $functions);
+        $this->assertSame([], $functions);
+    }
+
+    public function testGetFunctionParameterTypesShouldReturnTypesArray()
+    {
+        $functions = (new ReflectionTools)->getFunctionParameterTypes(new \ReflectionFunction('Brick\Reflection\Tests\reflectedParameterFunc'));
+
+        $this->assertCount(1, $functions);
+        $this->assertSame('string', $functions['arg'][0]);
```

Comment:
Can you test the actual array returned with `assertSame()` instead of these 2 lines?